kolibri_explore_plugin/collectionviews.py

```diff
@@ -27,6 +28,12 @@
 
 logger = logging.getLogger(__name__)
 
+COLLECTION_URL_TEMPLATE = (
+    "https://raw.githubusercontent.com/endlessm"
+    "/endless-key-collections/main/json"
+    "/{grade}-{name}.json"
+)
```

Comment:
This seems pretty dangerous to make this use the `main` HEAD commit at runtime. If someone commits a mistake to `main` or is in the process of refactoring the collections then all apps are broken immediately. I think if we want to move in this direction then we should make releases of endless-key-collections and use released artifacts.

Reply:
Yeah. @pwithnall started creating releases already. There is a ZIP file with all the source. I wonder if the JSON files could be separate release artifacts so they can be requested directly. Otherwise this code should request the ZIP and unpack it.

Reply:
Okey. The other improvement I'd do is allow changing the URL in an `options.ini` plugin setting. That way the same app build will be able to test different collections. It can be useful for content curation and testing (passing a mock collection with fewer content to avoid long download times).

Reply:
Another option which doesn't require releases and artifacts but is a little safer is to have a `stable` branch in endless-key-collections that we occasionally fast forward to a known good point on master. Then you can continue using the direct github links with more confidence that it won't randomly break.

Another thing to consider is using github pages since I expect that would scale a lot better than the raw links. We could arrange that the pages branch is only updated on release.

Reply:
The collection JSON is now published to https://endlessm.github.io/endless-key-collections per https://github.com/endlessm/endless-key-collections/pull/29, so I think you should change the base URL in this PR. When new tags are made in endless-key-collections, the published JSON will automatically be updated.